test_files/protected/protected.ts

```diff
@@ -0,0 +1,13 @@
+/** This test checks that we emit @private/@protected where necessary. */
```

Comment:
nit: should you escape the `\@private`? I never know how escaping even works in JSDoc though...